hack/tools/.golangci.yml

```diff
@@ -0,0 +1,24 @@
+run:
+  timeout: 2m
+
+linters:
+  disable-all: true
+  enable:
+    - deadcode
+    - errcheck
+    - gochecknoinits
+    - gofmt
+    - gosimple
+    - govet
+    - ineffassign
+    - misspell
+    - staticcheck
+    - structcheck
+    - typecheck
+    - unparam
+    - unused
+    - varcheck
+    - golint
+    - scopelint
```

Comment:
ideally the comments:
```
# default golangci-lint lints

...

# additional lints

...
```

from the shell configuration should be preserved in the yaml.